src/addons/dragAndDrop/WeekWrapper.js

```diff
@@ -79,42 +61,34 @@ class WeekWrapper extends React.Component {
     this.setState({ segment })
   }
 
-  handleMove = ({ x, y }, node, draggedEvent) => {
+  handleMove = (point, bounds, draggedEvent) => {
```

Comment:
All the changes in this file are hard to follow (though i'm sure correct and needed). What is is going on here, and how should test that it works?

Reply:
Purely idiomatic: all the other event handlers had the `(point, bounds)` signature (as does the similar code in `EventContainerWrapper`) so I was just conforming the idiom.

BTW, if you look closely, you'll see that `WeekWrapper` and `EventContainerWrapper` are in DIRE need of being DRY'ed out -- they basically do the same exact thing, with the main difference being the orientation of the event (horizontal or vertical). I spent a few days on that, but had to abandon due to time.

Reply:
... oh, and the other changes (sorry, that's probably what you were asking about) - were all about making the date handling correct during drags - which had a number of probs, some of which might have been introduce with the (required) upgrade to date-arithmetic (which was out-of-date and causing some probs).

Happy to answer in more detail about these edits although my memory is fading a bit on this.

Reply:
testing: I think the only way is via storybook/examples -- the code seems too intertwined IMHO for proper testing (definitely a good should a rewrite occur!)

Reply:
fine to defer the testing if there isn't a good way to write a unit test. I was thinking of the common-ish case where we fix one of these date bounds issues and introduce another one, partly because the calculations have no tests asserting their current behavior so easy to break stuff!